Sources/Vapor/Error/Abort.swift

```diff
@@ -12,11 +12,24 @@ public struct Abort: AbortError, DebuggableError {
     ///
     /// Set type to '.permanently' to allow caching to automatically redirect from browsers.
     /// Defaulting to non-permanent to prevent unexpected caching.
+    @available(*, deprecated, renamed: "redirectTo")
     public static func redirect(to location: String, type: RedirectType = .normal) -> Abort {
         var headers: HTTPHeaders = [:]
         headers.replaceOrAdd(name: .location, value: location)
         return .init(type.status, headers: headers)
     }
+    
+    /// Creates a redirecting `Abort` error.
+    ///
+    ///     throw Abort.redirectTo("https://vapor.codes")"
+    ///
+    /// Set type to '.permanently' to allow caching to automatically redirect from browsers.
+    /// Defaulting to non-permanent to prevent unexpected caching.
+    public static func redirectTo(_ location: String, type: Redirect = .normal) -> Abort {
```

Comment:
Have to say I'm not a fan of the new API - how about
```suggestion
    public static func redirect(to location: String, redirectType: Redirect = .normal) -> Abort {
```
Fits better with the Swift API guidelines, looks better and makes the redirect type clearer

Reply:
I prefer this too, the only issue is that if we keep around the deprecated `redirect(to, type)`, then when using the default `redirectType` argument, the call is ambiguous. That is, calling `request.redirect(to: "/")`, it wouldn't know whether it was the old `redirect(to, type)` or the new `redirect(to, redirectType)`, and will throw a compilation error.

Reply:
We can actually be clever/sneaky here - the old API doesn't need a default value. That migrates everyone to the new API unless they're specifying a redirect type without a breaking change

Reply:
Ohhh, that's so smart! I love it! I've made this change